NFIQ2/NFIQ2Algorithm/src/nfiq2/nfiq2impl.cpp

```diff
@@ -170,13 +170,13 @@ NFIQ2Algorithm::Impl::computeQualityFeatures(
 	FingerJetFXFeature fjfxFeatureModule(bOutputSpeed, speedValues);
 	// this module returns the FJFX minutiae template to be used in other
 	// modules
-	unsigned char
-	    fjfxTemplateData[4096]; // allocate 4KB of memory for template
-	size_t fjfxTemplateSize = sizeof(fjfxTemplateData);
+
+	std::unique_ptr<FRFXLL_Basic_19794_2_Minutia[]> minutiaData {};
+	unsigned int minutiaCount;
```

Comment:
Suggest initialization.
```suggestion
	unsigned int minutiaCount{0};
```